runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -513,7 +513,7 @@ static void AM_priv_bcast(gasnet_token_t token, void* buf, size_t nbytes) {
 
 static void AM_priv_bcast_large(gasnet_token_t token, void* buf, size_t nbytes) {
   priv_bcast_large_t* pblp = buf;
-  chpl_memcpy((char*)chpl_private_broadcast_table[pblp->id]+pblp->offset, pblp->data, pblp->size);
+  chpl_memmove((char*)chpl_private_broadcast_table[pblp->id]+pblp->offset, pblp->data, pblp->size);
```

Comment:
This can't overlap. Same as above: the target is the address of a Chapel global variable and the source is within a GASNet-managed incoming message buffer.